.github/workflows/run-tests-windows.yml

```diff
@@ -45,7 +46,84 @@ jobs:
           $sharedDirs   = @("${{ join(matrix.gc.include, '", "') }}") | %{Join-Path $gcRoot $_}
           $compiledDirs = @($gcDir) + $sharedDirs
           $includes     = $sharedDirs | %{"-I" + $_}
-          Get-ChildItem -Path $compiledDirs -Include *.c -Recurse -File `
+          Get-ChildItem -Path $compiledDirs -Filter *.c -Recurse -File `
           | ForEach-Object {clang -std=gnu11 -Wno-override-module $includes -c "$_"}
-          Get-ChildItem -Path $compiledDirs -Include *.cpp -Recurse -File `
-          | ForEach-Object {clang++ -std=c++14 -Wno-override-module $includes -c "$_"}
\ No newline at end of file
+          Get-ChildItem -Path $compiledDirs -Filter *.cpp -Recurse -File `
+          | ForEach-Object {clang++ -std=c++14 -Wno-override-module $includes -c "$_"}
+
+  compile-non-gc:
```

Comment:
Is there any reason why these are not simply more steps in the first job?

Reply:
First job defined matrix for different GCs, this way we compile another source only once.